packages/replay-internal/test/integration/sendReplayEvent.test.ts

```diff
@@ -406,6 +410,8 @@ describe('Integration | sendReplayEvent', () => {
     // Replay has stopped, no session should exist
     expect(replay.session).toBe(undefined);
     expect(replay.isEnabled()).toBe(false);
+    expect(onError).toHaveBeenCalledTimes(5);
```

Comment:
why is it 5?

Reply:
It gets called each time replay fails to send until it hits max